cli.js

```diff
@@ -34,9 +35,15 @@ var cli = meow({
 		'test.js test-*.js test/*.js'
 	]
 }, {
-	string: ['_']
+	string: ['_'],
+	boolean: ['fail-fast']
 });
 
+// set env variable, so that forked process can detect the flag
+if (cli.flags.failFast) {
+	process.env.AVA_FAIL_FAST = true;
```

Comment:
Would be cleaner to pass this as a command-line option when forking the process.